services/common/src/components/projects/ArchivedDocumentsSection.tsx

```diff
@@ -7,12 +7,15 @@ import { useFeatureFlag } from "@mds/common/providers/featureFlags/useFeatureFla
 import DocumentTable from "../documents/DocumentTable";
 import { renderCategoryColumn } from "../common/CoreTableCommonColumns";
 import { CATEGORY_CODE } from "../..";
+import { ColumnType } from "antd/es/table";
 
 interface ArchivedDocumentsSectionProps {
   documents: MineDocument[];
   titleLevel?: 1 | 2 | 3 | 4 | 5;
   href?: string;
   showCategory?: boolean;
+  additionalColumns?: ColumnType<MineDocument>[];
+  documentColumns?: ColumnType<MineDocument>[];
```

Comment:
Wait- I see these props being added, but they're not being used by the component.

Reply:
I think you can use the `showCategory` instead of additionalColumns in this use case (so long as the documents get their category assigned...) and then just pass the documentColumns to DocumentTable.

Although- if it's showing the columns correctly as is, then you probably don't need them!

Reply:
ArchivedDocumentsSection still doesn't use documentColumns. Does it actually need it at this point?